torch/_inductor/kernel/mm_scaled.py

```diff
@@ -243,6 +465,33 @@ def scaled_mm_options(  # type: ignore[no-untyped-def]
 add_layout_constraint(aten._scaled_mm.default, constrain_to_fx_strides)
```

Comment:
Sadly this just does stride order lol. in any case, that should be fixed separately.

Reply:
I always forget this